Pipfile

```diff
@@ -26,9 +26,10 @@ msgpack = "*"
 aiohttp = "==3.8.2"
 flask = "*"
 requests = "*"
+urllib3 = ">=1.26.16,<2" # because of eth-ape
```

Comment:
Was this change and the one below necessary? Or could the version of `eth-ape` just be limited to "<=0.6.19".

I'd much rather limit a dev-dependency that a regular dependency.

Reply:
Yes, I agree. We limited eth-ape to <=0.6.19 and it worked for urllib3 limitation. But we still needed to limit py-evm to <0.8 because of evm-trace

Reply:
Ah ok 👍